lib/matplotlib/tests/test_figure.py

```diff
@@ -557,6 +557,24 @@ def test_savefig_pixel_ratio(backend):
     assert ratio1 == ratio2
 
 
+@image_comparison(['transparent_background'],
+                    extensions=['png'], savefig_kwarg={'transparent': True},
+                    remove_text=True)
+def test_savefig_transparent():
+    # create two transparent subfigures with corresponding transparent inset
+    # axes. the entire background of the image should be transparent.
+    fig = plt.gcf()
+    gs = fig.add_gridspec(3, 3)
+    fig.add_subfigure(gs[0, 0])
+    fig.add_subplot(gs[0, 0])
+    ax = plt.gca()
+    ax.inset_axes([.1, .2, .3, .4])
+    fig.add_subfigure(gs[0:2, 1])
+    fig.add_subplot(gs[0:2, 1])
+    ax2 = plt.gca()
+    ax2.inset_axes([.1, .2, .3, .4])
```

Comment:
I think you can add `ax.spines[:].set_visible(False); ax.set(xticks=[], yticks=[])` to all axes instantiations, which effectively makes them invisible (and thus the entire savefig should be invisible).  Then you can check_figures_equal with a fully empty plot.